### ISSUE TITLE
[feature request] detect empty hooks

### ISSUE THREAD

**Issue by BridgeAR** [lo1tuma/eslint-plugin-mocha#409]
Especially with LLMs being around now, having rules for things like that would likely be a good idea.

E.g., 

```js
after(async () => {})
```

**Comment by lo1tuma** [comment]
The Eslint core rule [`no-empty-function`](https://eslint.org/docs/latest/rules/no-empty-function) should be able to catch this.
